GVFS/GVFS.UnitTests.Windows/Windows/Mock/MockInstallerPreRunChecker.cs

```diff
@@ -72,9 +72,9 @@ protected override bool IsUnattended()
             return this.FakedResultOfCheck(FailOnCheckType.UnattendedMode);
         }
 
-        protected override bool IsBlockingProcessRunning(out HashSet<string> processes)
+        protected override bool IsBlockingProcessRunning(out List<string> processes)
         {
-            processes = new HashSet<string>();
+            processes = new List<string>();

```

Comment:
Is there a reason why you changed from `List` to `HashSet`? or was this just a general change?

Reply:
I can switch it back to HashSet.  Since we give the pid they're unique.